Sources/NetworkProtection/Notifications/NetworkProtectionNotification.swift

```diff
@@ -21,18 +21,6 @@ import Common
 
 #if os(macOS)
 
-fileprivate extension Bundle {
```

Comment:
This is called from iOS so, for now, needs to be outside of the macOS compiler-flagged code